buildSrc/build.gradle.kts

```diff
@@ -215,4 +211,15 @@ fun Project.applyKotlinProjectConventions() {
             experimentalWarning.set(false)
         }
     }
+
+    configure<org.jlleitschuh.gradle.ktlint.KtlintExtension> {
+        // TODO:kotlin-dsl unignore KtLint failures
+        // we would like to only exclude the generated-sources from the
+        // verification but unfortunately the filter doesn't seem to
+        // have an effect so :/
+        ignoreFailures.set(true)
+        filter {
+            exclude("**/generated-sources/**")
+        }
+    }
```

Comment:
😢 Womp womp. CC: @Tapchicoma

@bamboo Have you tried the latest version of Ktlint-Gradle?

Reply:
@JLLeitschuh, ktlint-gradle 0.7.1 has the same problem

Reply:
Ah, the exclude patterns only apply to the path relative to each source set srcDir. The `**/generated-sources/**` pattern tries to match the path up that dir, that's why it didn't work.

Reply:
Fixed in bdd3008e459